Packages/com.unity.inputsystem/InputSystem/Plugins/Switch/NPad.cs

```diff
@@ -346,7 +346,7 @@ public static NPadDeviceIOCTLOutputCommand Create()
     }
 #endif
 
-#if UNITY_EDITOR || UNITY_STANDALONE_OSX || UNITY_STANDALONE_WIN || UNITY_WSA
+#if UNITY_EDITOR || UNITY_STANDALONE || UNITY_WSA
```

Comment:
Think this should be the opposite. Don't think we're support the Switch Pro controller this way on Linux ATM. Probably makes more sense to have the entire Switch stuff read `UNITY_STANDALONE_WIN || UNITY_STANDALONE_OSX` rather than `UNITY_STANDALONE`.

Reply:
ok fixed, went to the other version.  Now consistent.